Update Offline import steps (#928)
filecoin-project/filecoin-docs: docs/store/lotus/very-large-files.md

1. Use the Lotus client to generate a CAR file of the input without importing:

```sh

> lotus client generate-car <inputPath/OriginalFile> <outputPath/Name.car>

Reviewer: ```suggestion
lotus client generate-car <input_path/filename> <output_path/data.car>
```

### Calculate the datasize

1. Take the final car size `ls -l data.car`
2. Round it up to the nearest power of 2. This is your padded piece size.
3. Divide by 128, multiply by 127. this is your unpadded piece size.

> 4. Alternative  A. Go to [Wolfram Alpha](https://www.wolframalpha.com).
> 5. Alternative  B. Enter `x = SIZE_IN_BYTES; 127*( 2^( ceil( log2( ceil ( x /127 ) ) ) ) )` and changing `SIZE_IN_BYTES` to your value.
> 6. Alternative  C. You solution can be found under **Substitution**:
> 
> 
> ![](./images/very-large-files/wolfram-alpha.png)

Reviewer: ```suggestion

You can also use WolframAlpha to get your datasize:

1. Go to [Wolfram Alpha](https://www.wolframalpha.com).
1. Enter `x = SIZE_IN_BYTES; 127*( 2^( ceil( log2( ceil ( x /127 ) ) ) ) )`, replacing `SIZE_IN_BYTES` to your value.
1. Your solution can be found under **Substitution**
    
    ![](./images/very-large-files/wolfram-alpha.png)
```

```sh
lotus client deal --manual-piece-cid=CID --manual-piece-size=datasize <Data CID> <miner> <price> <duration>
```

> Data CID is provided using this method https://docs.filecoin.io/store/lotus/store-data/#importing-data

Reviewer: ```suggestion
Your Data CID is provided when you import our data into Lotus using `lotus client import`.
```
